UnitTests/Utilities/GeometryModelRenderer/src/VisualTest.h

```diff
@@ -22,6 +22,7 @@
 
 #include <namespace.h>
 
+#include <OffGeometryModelRenderer.h>
 #include <IfcGeometryModelRenderer.h>
```

Comment:
this is not needed in the `VisualTest` class, but their corresponding children

Reply:
Resolved: 59b6a3f